app/helpers/countries_helper.rb

```diff
@@ -7,6 +7,12 @@ def is_japan?
     @country && @country.iso_3 == "JPN"
   end
 
+  def has_restricted_sites?
+    restricted_iso3 = ["RUS", "EST", "CHN", "GBR"]
+    
+    return restricted_iso3.include? @country.iso_3
```

Comment:
`return` is not strictly necessary here, the function will automatically return the result of the latest line of code.
 
Moreover, you could include the `@country` bit here and call this only function in the view